src/main/java/core/basesyntax/services/ReportCreaterService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.services;
+
+import java.util.Map;
+
+public interface ReportCreaterService {
+    String createReport(Map<String, Integer> parsedMap);
```

Comment:
```suggestion
    String createReport();
```
Pass Storage in constructor